app/src/test/java/com/example/concordia_campus_guide/ModelTests/ShuttleTest.java

```diff
@@ -56,4 +56,21 @@ public void testShuttles() {
         List<Shuttle> actual = shuttles.getShuttles();
         assertEquals(expected, actual);
     }
+
+    @Test
+    public void setShuttleIdTest() {
+        int id = 1;
+        shuttle.setShuttleId(id);
+        assertEquals(id, shuttle.getShuttleId());
+    }
+
+    @Test
+    public void toStringTest() {
+        shuttle.setCampus("SGW");
+        shuttle.setDay(Arrays.asList("Monday"));
+        shuttle.setTime(7.05);
```

Comment:
Creating a local variable with this value (like you did with the id in setShuttleIdTest) and using that var (e.g `String expected = "Campus: SGW, Day: [Monday], time: " + timeValue + "7.05\n";`) in the expected value could be useful for storing that random ""constant" in a common place. For this PR, this can be a little nitpicky so no need to waste too much time implementing this change, but it'd be nice to remember this in the future.